src/customFunctions/customFetch.ts

```diff
@@ -0,0 +1,18 @@
+import { bearerTokenAuthorization } from "./../configs/bearerTokenAuthorization";
+
+export const customFetch = async (
+  input: RequestInfo,
+  init: RequestInit = {}
+) => {
+  init.headers = bearerTokenAuthorization.headers;
+  const response = await fetch(input, init);
+
+  if (!response.ok) {
+    throw new Error(`Response status code: ${response.status}`);
+  }
+
+  return {
+    fullResponse: response.clone(),
+    json: await response.json(),
+  };
```

Comment:
my ve storemodu naproramovali fetch, že vrací:
```
return [json, response]
```

je to mikro drobnost, ale je hezké, že vlastně tu response v 90% requestů nepotřebuješ, tak můžeš pak dát jen:
```
const [data] = majFeč('htptptsps;?//...')
```

což je podle mě hezké minimalistické rozhraní